### PR TITLE
chore: remove more cx ffs

### PR DESCRIPTION
**Pull request by MounirDhahri**
The type of this PR is: **Chore**

This PR solves [CX-3417]

### Description

Remove uploading flow-related feature flags
Remove upcoming auction results feature flag

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CX-3417]: https://artsyproduct.atlassian.net/browse/CX-3417?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormArtistStep.tsx [hunk 1]
```diff
@@ -126,38 +122,32 @@ export const MyCollectionArtworkFormArtistStep: React.FC<MyCollectionArtworkForm
             >
               {query}
             </Text>
-            “ on Artsy.{" "}
-            {!!enablePersonalArtists &&
-              "You can add their name in the artwork details."}
+            “ on Artsy. "You can add their name in the artwork details."
```
Comment (olerichter00) [inline src/Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormArtistStep.tsx:125]:
```suggestion
            “ on Artsy. You can add their name in the artwork details."
```

**Review by olerichter00 — CHANGES_REQUESTED** [review]

File: src/Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormArtistStep.tsx (hunk 1 above)
Comment (MounirDhahri) [reply src/Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormArtistStep.tsx:125]: thanks- I will fix that

**Comment by MounirDhahri** [comment]
Will merge this before merge conflicts

File: src/Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormArtistStep.tsx (hunk 1 above)
Comment (olerichter00) [reply src/Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormArtistStep.tsx]: Let's check this on staging when it's deployed. I'm still quite confused by all the `"`'s :)

**Pull request merged**